os_migrate/roles/conversion_host/defaults/main.yml

```diff
@@ -27,7 +27,8 @@ os_migrate_conversion_keypair_private_path: "{{ os_migrate_data_dir }}/conversio
 os_migrate_conversion_link_keypair_private_path: "{{ os_migrate_data_dir }}/conversion/link-ssh.key"
 
 os_migrate_conversion_host_deploy_timeout: 600
-os_migrate_conversion_host_name: os_migrate_conv
+os_migrate_src_conversion_host_name: os_migrate_conv
+os_migrate_dst_conversion_host_name: os_migrate_conv
```

Comment:
Can we also use different defaults? `os_migrate_conv_src` and `os_migrate_conv_dst` perhaps? This may "fix" it by default for people who use Satellite?

Reply:
sure thing